ee/codegen/src/__test__/__snapshots__/vellum-variable.test.ts.snap

```diff
@@ -31,7 +31,7 @@ exports[`VellumVariableField > JsonVellumVariable snapshot 1`] = `
 `;
 
 exports[`VellumVariableField > NullVellumVariable snapshot 1`] = `
-"test: None
+"test: Optional[None]
```

Comment:
[nit] This should likely just be `None`. `Optional[None]` resolves to `None | None` which is just `None`